.github/workflows/pathogen-repo-ci.yaml

```
@@ -63,6 +63,16 @@ on:
           - conda
         required: false
 
+      build-dir:
+        description: >-
+          The `BUILD_DIR` for `nextstrain build $BUILD_DIR`.
+          Example data is expected to be in `$BUILD_DIR/example_data/`
+          Defaults to the root of the workflow repo.
+          Directory string must not end in a slash.
```

_(non-blocking)_

Some minor wording changes. Notably, I'd avoid using an example variable name. If you want to template things, I'd suggest using the input name as-is (e.g. `nextstrain build <build-dir>`).

```suggestion
          The directory used for `nextstrain build `.
          If provided, example data should be in a `example_data` directory relative to this directory.
          Defaults to the root of the repository.
          Directory string must not end in a slash.
```